gufe/settings/models.py

```diff
@@ -85,7 +87,7 @@ class Settings(SettingsBaseModel):
     settings_version: int = 0
     forcefield_settings: OpenMMSystemGeneratorFFSettings
     thermo_settings: ThermoSettings
-    protocol_settings: Union[ProtocolSettings, None]
+    protocol_settings: type[ProtocolSettings]
```

Comment:
Also `Settings` must include the Protocol specific section.  Again, I couldn't think of a good example of when you'd not want to have Protocol specific settings.  If there was one, you could always define an empty dict.